.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
+# Ignore CHANGELOG.md files
+/packages/**/CHANGELOG.md @ghost
```

Comment:
Nice hack 😄 I think that's acceptable 👌 

```suggestion
# Ignore CHANGELOG.md files to prevent Danger.js from triggering code owner reviews
/packages/**/CHANGELOG.md                       @ghost

```